weblate/trans/models/component.py

```diff
@@ -154,6 +155,23 @@
 
 LOCKING_ALERTS = {"MergeFailure", "UpdateFailure", "PushFailure", "ParseError"}
 
+BITBUCKET_GIT_REPOS_REGEXP = [r"([a-z]+):\/\/([^/]*)\/([^/]*)", r"git@([^/]*):([^/]*)"]
```

Comment:
This seems to be wrong, sample URLs:

* `https://weblate@bitbucket.org/sasajovanovic/adbukalng.git`
* `git@bitbucket.org:sasajovanovic/adbukalng.git`